libraries/AP_Mount/AP_Mount.h

```diff
@@ -73,7 +77,8 @@ class AP_Mount
         Mount_Type_SoloGimbal = 2,      /// Solo's gimbal
         Mount_Type_Alexmos = 3,         /// Alexmos mount
         Mount_Type_SToRM32 = 4,         /// SToRM32 mount using MAVLink protocol
-        Mount_Type_SToRM32_serial = 5   /// SToRM32 mount using custom serial protocol
+        Mount_Type_SToRM32_serial = 5,   /// SToRM32 mount using custom serial protocol
```

Comment:
One whitespace too much.
Please add a Visca comment

Reply:
I'm sorry, I should have noticed that before pushing. Fixed.